lando_cli/cli.py

```diff
@@ -164,16 +173,47 @@ def wait_for_job_completion(
     return result
 
 
-def submit_to_lando(config: Config, repo_name: str, actions: list[dict]):
+def display_relbranch_tracking_warning(branch_name: str):
+    """Display a warning about the new RelBranch."""
+    click.secho(
+        (
+            f"\nNote: The new RelBranch `{branch_name}` was created or updated, "
+            "but your local copy does not yet track it."
+        ),
+        fg="yellow",
+        bold=True,
+    )
+    click.echo(f"Fetch the latest changes on the relbranch:\n")
+    click.echo(f"  $ git fetch origin {branch_name}\n")
+    click.echo("Ensure you have the local branch checked out:\n")
+    click.echo(f"  $ git switch {branch_name}\n")
+    click.echo("Update your local branch to point to the official branch:\n")
+    click.echo(f"  $ git reset origin/{branch_name}\n")
+    click.echo("Set your local branch to track the official branch:\n")
+    click.echo(f"  $ git branch --set-upstream-to=origin/{branch_name} {branch_name}\n")
+    click.echo("This will give you a fresh checkout of the landed commits.\n")
```

Comment:
nit: could put these in a list then do a `click.echo` while iterating, instead of repeating the command.

Reply:
I have a bunch of cleanups I'd like to do as follow-ups, this is one of them. :)